.github/workflows/ci.yml

```diff
@@ -35,12 +17,29 @@ jobs:
     name: Build Slicer
     runs-on: ubuntu-latest
     steps:
+      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # v2.11.1
+        id: changes
+        with:
+          filters: |
+            paths-to-include:
```

Comment:
```suggestion
            paths-to-include:
              - ".github/workflows/ci.yml"
              - ".github/actions/slicer-build/**"
```

Reply:
Would it be possible to move the improvement to `.github/workflows/ci.yml` into its own pull request ?

Reply:
Here you go https://github.com/Slicer/Slicer/pull/7062